BTCPayServerDockerConfigurator/Views/Configurator/Additional/TorRelaySettings.cshtml

```diff
@@ -7,7 +7,7 @@
     </label>
     <div class="card-body">
         <p class="card-text">
-            Tor relays are also referred to as "routers" or "nodes." They receive traffic on the Tor network and pass it along. This sets up a non-exit tor relay alongside your BTCPay. <br/><a href="https://www.eff.org/torchallenge/faq.html" target="_parent">Please read the legal implications of running a tor relay</a> and <a href="https://trac.torproject.org/projects/tor/wiki/TorRelayGuide#RelayRequirements" target="_blank">what resources are used to operate the relay.</a>
+            Tor relays are also referred to as "routers" or "nodes." They receive traffic on the Tor network and pass it along. This sets up a non-exit tor relay alongside your BTCPay. <br/><a href="https://www.eff.org/torchallenge/faq.html" target="_blank">Please read the legal implications of running a tor relay</a> and <a href="https://trac.torproject.org/projects/tor/wiki/TorRelayGuide#RelayRequirements" target="_blank">what resources are used to operate the relay.</a>
```

Comment:
@Zaxounette the link is dead, I updated the same link to [this one](https://github.com/btcpayserver/btcpayserver-docker/pull/380) in the docker repo recently

Reply:
```suggestion
            Tor relays are also referred to as "routers" or "nodes." They receive traffic on the Tor network and pass it along. This sets up a non-exit tor relay alongside your BTCPay. <br/><a href="https://community.torproject.org/relay/community-resources/eff-tor-legal-faq/" target="_blank">Please read the legal implications of running a tor relay</a> and <a href="https://trac.torproject.org/projects/tor/wiki/TorRelayGuide#RelayRequirements" target="_blank">what resources are used to operate the relay.</a>
```

Reply:
Thanks for the heads up @britttttk =)